stix_shifter/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -4,6 +4,7 @@
 
 from . import observable
 from stix2validator import validate_instance, print_results
+from stix_shifter.src.modules.qradar import qradar_utils
```

Comment:
json to Stix translator not supposed to import QRadar or any other module,
that's why I was suggesting doing it via a callback